src/platform/site-wide/user-nav/components/SearchMenu.jsx

```diff
@@ -244,7 +251,7 @@ export class SearchMenu extends React.Component {
     } = this;
 
     const highlightedSuggestion =
-      'suggestion-highlighted vads-u-background-color--primary-alt-light vads-u-color--gray-dark vads-u-margin-x--0 vads-u-margin-top--0p5 vads-u-margin-bottom--0  vads-u-padding--1 vads-u-width--full vads-u-padding-left--2';
+      'suggestion-highlighted vads-u-background-color--primary-alt-light .vads-u-color--gray-dark vads-u-margin-x--0 vads-u-margin-top--0p5 vads-u-margin-bottom--0  vads-u-padding--1 vads-u-width--full vads-u-padding-left--2';
```

Comment:
Did you mean to add `.vads-u-color--gray-dark` in here? Maybe `vads-u-color--gray-dark`?

Reply:
ahh thats a oopsie from when I was rolling things back!